### PR TITLE
feat(workflow): Dynamic issue counts followup

### PR DESCRIPTION
**Pull request by taylangocmen**
In this PR:
- Changed the order in the tooltip: filtered -> time range -> global
- "Frequency" -> "Events" in the sort dropdown
- "Within" -> "In" in the count tooltip
- Change Default sort order for issue stream to "14d"
- Round counts in the tooltip similar to issue stream
- Fix a bug where if a tag value had spaces and quotations queryObj function stripped them and discover links didn't work
- Aligned count tooltip content to be perfectly aligned table

Resolves:
[WOR-216](https://getsentry.atlassian.net/browse/WOR-216)
[WOR-212](https://getsentry.atlassian.net/browse/WOR-212)

### CODE REVIEW

File: src/sentry/static/sentry/app/views/issueList/overview.jsx
Comment (markstory) [inline src/sentry/static/sentry/app/views/issueList/overview.jsx:221]: This looks like it would change for all users not just those that are exposed to the new issue counts. Is that ok?

**Review by iProgramStuff — COMMENTED** [review]
This looks good to me..but I'd wait for a second approval from someone more experienced in our frontend

File: src/sentry/static/sentry/app/components/stream/group.jsx
Comment (billyvg) [inline src/sentry/static/sentry/app/components/stream/group.jsx:144]: Do you know if this is logic that's used elsewhere in the app? cc @markstory

File: src/sentry/static/sentry/app/components/stream/group.jsx
Comment (billyvg) [inline src/sentry/static/sentry/app/components/stream/group.jsx:378]: Is this still needed?

File: src/sentry/static/sentry/app/views/issueList/overview.jsx
Comment (taylangocmen) [reply src/sentry/static/sentry/app/views/issueList/overview.jsx:221]: I'll revert these changes, I have another pr I'll put up that has more substantial changes to these

File: src/sentry/static/sentry/app/components/stream/group.jsx
Comment (taylangocmen) [reply src/sentry/static/sentry/app/components/stream/group.jsx:378]: It's not needed but I thought maybe keeping the styled component names would be nice, but actually cleaner code would be nicer, I'll remove this

File: src/sentry/static/sentry/app/components/stream/group.jsx
Comment (taylangocmen) [reply src/sentry/static/sentry/app/components/stream/group.jsx:144]: There are other locations that link to discovery with modified query but this is a unique use case. Maybe I can make a `utils/stream` function that picks out tags from query objects, but that felt like an overkill

Comment (markstory) [reply src/sentry/static/sentry/app/components/stream/group.jsx:143]: We have a few places that manipulate queries. Most of those use the tokenizeSearch module. The QueryResult object in that module lets you add and remove tags and gives you a better string conversion API that handles quoting better. You'd still need to drop the `is` condition though.

**Review by billyvg — APPROVED** [review]
Can we follow up and see if it makes sense to use `tokenizeSearch`?

**Comment by taylangocmen** [comment]
> Can we follow up and see if it makes sense to use `tokenizeSearch`?

Thanks, I'll follow up, you can keep an eye on this one:
https://getsentry.atlassian.net/browse/WOR-237

**Pull request merged**
